<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>348</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>American Typewriter</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>331</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page0_Shop">
     <widget class="QLabel" name="labelFanta">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fanta</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumFanta">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumSprit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCock">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>51</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSprit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sprite</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonFanta">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>51</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPepsi">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>51</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonViewCart">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Founders Grotesk</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>View Cart</string>
      </property>
     </widget>
     <widget class="QLabel" name="Title">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>American Typewriter</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Bremen Store</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCock">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cock</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumPepsi">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPepsi">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pepsi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSprit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>51</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumCock">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page1_ItemList">
     <widget class="QLabel" name="labelItem">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>American Typewriter</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Items in Card</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetofItem">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>181</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelTotalItem">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTotalPriceInCard">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPay">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Founders Grotesk</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackShopPage">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Founders Grotesk</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2_Payment">
     <widget class="QLabel" name="label_pyment">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>American Typewriter</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Payment</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonConfirm">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Founders Grotesk</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTotalPrice">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTotalPyment">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCardNum">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Skia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Card Number:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>104</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditCarNum">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>104</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3_Thank">
     <widget class="QLabel" name="labelNewShop">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>American Typewriter</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Thank you</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNewShop">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Founders Grotesk</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Go shoping</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditShow">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>321</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
